common/app/Panes/quotes.js

```diff
@@ -0,0 +1,73 @@
+const quotes = [
+  {
+    quote: 'Never, never, never, never, never, never give up.',
+    author: 'Winston Churchill'
+  },
+  {
+    quote: 'The pessimist sees difficulty in every opportunity.' +
+    'The optimist sees opportunity in every difficulty.',
+    author: 'Winston Churchill'
+  },
+  {
+    quote: 'Twenty years from you you will be more disappointed by the ' +
+    'things that you didn\'t do than by the ones you did do. So throw ' +
```

Comment:
These concatenated strings should be indented by (one more tab || 2 more spaces), I have no preference which you use.

Same comment for all instances in this file :+1: